lib/takeaway.rb

```diff
@@ -0,0 +1,36 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Takeaway
+  attr_reader :order_klass, :order, :order_time
+
+  def initialize(order_klass)
+    @order_klass = order_klass
+    @order = nil
+  end
+
+  def new_order(menu_klass)
+    @order = order_klass.new(menu_klass)
+  end
+
+  def place_order(order, quantity, total, number)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)